Source/Plugins/Core/com.equella.core/resources/lang/i18n-resource-centre.properties

```
@@ -80,6 +80,7 @@
 /com.tle.core.entity.services.migration.v20192.unknownuser.alluser=Create a new table for all users
 /com.tle.core.entity.services.migration.v20202.facetedsearch.classification=Create a new table for faceted search classification
 /com.tle.core.entity.services.migration.v20202.removelastknownuserconstraint=Remove last known user composite constraint (username and institution ID)
+/com.tle.core.entity.services.migration.v20202.indexing.errored=Add column to attachment table to allow indexer to skip individual attachments that have timed out.
```

Be worthwhile to keep wording around it generic. That way if in the future we have other failure modes, we're all set.

```suggestion
/com.tle.core.entity.services.migration.v20202.indexing.errored=Add column to attachment table to allow indexer to skip individual attachments that have failed.
```